repo/manifest/committed_manifest_manager.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/kopia/kopia/repo/content/index"
 )
 
+const maxManifestsPerContent = 1000000
```

Comment:
Curious, how did we get to this number? Is this assuming some average size per manifest entry from data we have?